R/suffix.R

```diff
@@ -142,8 +142,9 @@ suffix_extract <- function(domains, suffixes = NULL){
         stop("Expected column named \"suffixes\" in suffixes data.frame")
       }
     }
-    suffix_load(suffixes)
   }
+  suffix_load(suffixes)
```

Comment:
Mmn. I think I can come up with something; suggest dropping to just the usage fix and I'll fill in the rest!